docs/modules/query/pages/continuous-query-cache.adoc

```diff
@@ -82,6 +82,16 @@ latter is mostly explained in the previous section. The parent configuration ele
 is `<query-caches>` which should be placed within your `<map>` configuration. You can
 create your query caches using the  `<query-cache>` sub-element under `<query-caches>`.
 
+NOTE: Query Cache returns/uses internal key object reference during
```

Comment:
I think this is too much information for the user.

How about this?

> If you use mutable keys in maps, consider setting the `serialize-keys` option to `true` to tell Hazelcast to clone the keys rather than use an internal reference. Otherwise, Hazelcast may store references to outdated versions of your keys.